From ec87cd9d493cd7195bca620314c7f4c65f6af4ca Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Mon, 9 Oct 2017 17:00:02 -0500
Subject: [PATCH] don't fail for non-existent --with-system-ssl-certs dir

Currently, pidgin will fail if a non-existent directory is
specified in --with-system-ssl-certs. As we use a /var directory,
it's created on the fly, hence we should warn instead of failing.

Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index b8acd2a..fafc278 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1757,7 +1757,7 @@ if ! test -z "$ssl_certificates_dir" ; then
 		AC_MSG_ERROR([--with-system-ssl-certs requires that a location is specified, eg. --with-system-ssl-certs=/etc/pki/tls/certs])
 	fi
 	if ! test -d "$ssl_certificates_dir" ; then
-		AC_MSG_ERROR([$ssl_certificates_dir does not exist, if this is the correct location please make sure that it exists.])
+		AC_MSG_WARN([$ssl_certificates_dir does not exist, using non-existent ssl directory! This may be an error for your build!])
 	fi
 	SSL_CERTIFICATES_DIR="$ssl_certificates_dir"
 fi
-- 
2.14.2

